crowd_anki/utils/constants.py

```diff
@@ -2,9 +2,10 @@
 
 UUID_FIELD_NAME = 'crowdanki_uuid'
 DECK_FILE_NAME = "deck"
-SETTINGS_FIELD_NAME = '_settings'
 DECK_FILE_EXTENSION = ".json"
 MEDIA_SUBDIRECTORY_NAME = "media"
+IMPORT_CONFIG_NAME = "import_config"
+CONFIG_EXTENSION = ".yaml"
```

Comment:
not sure if splitting these adds much value tbh

Reply:
:shrug: don't much mind

Reply:
Removed.